roles/netbootxyz/templates/disks/netboot.xyz.j2

```diff
@@ -20,14 +20,26 @@ iseq ${site_name} netboot.xyz || echo ${bold}${fg_whi}Powered by ${fg_gre}netboo
 prompt --key m --timeout 4000 Hit the ${bold}m${boldoff} key to open failsafe menu... && goto failsafe || goto dhcp
 
 :dhcp
-echo
 dhcp || goto netconfig
-isset ${next-server} && iseq ${filename} {{ site_name }}.kpxe && goto tftpmenu ||
-isset ${next-server} && iseq ${filename} {{ site_name }}-undionly.kpxe && goto tftpmenu ||
-isset ${next-server} && iseq ${filename} {{ site_name }}.efi && goto tftpmenu ||
-isset ${next-server} && iseq ${filename} {{ site_name }}-snp.efi && goto tftpmenu ||
-isset ${next-server} && iseq ${filename} {{ site_name }}-snponly.efi && goto tftpmenu ||
-isset ${next-server} && iseq ${filename} {{ site_name }}-arm64.efi && goto tftpmenu ||
+isset ${next-server} && isset ${proxydhcp/next-server} && goto choose-tftp || set tftp-server ${next-server} && goto load-custom-ipxe
+
+:choose-tftp
+# Load "proxy settings" from root server
+chain tftp://${next-server}/proxy-dhcp-vars.ipxe || echo Could not load definitions to resolve proxy DHCP conflict
+# Check if the proxy-dhcp-vars script has made any usable command about how to progress with a next-server and a proxy-next-server being set
+isset ${force_use_root_dhcp_settings} && iseq ${force_use_root_dhcp_settings} true && set tftp-server ${next-server} && goto load-custom-ipxe || echo -n
+isset ${force_use_root_dhcp_settings} && iseq ${force_use_root_dhcp_settings} false && set tftp-server ${proxydhcp/next-server} && goto load-custom-ipxe || echo -n
+prompt --key 0x107e --timeout 4000 DHCP proxy detected. Press ${bold}F5${boldoff} to apply setting from it && set tftp-server ${proxydhcp/next-server} || set tftp-server ${next-server}
```

Comment:
Switch to a character and shorten the output.  (just want to avoid function key complexities)
```
prompt --key p --timeout 4000 DHCP proxy detected. Press ${bold}p${boldoff} to boot from && set tftp-server ${proxydhcp/next-server} || set tftp-server ${next-server}
```